db/migrate/20160825213523_change_load_balancer_listener_port_to_port_range.rb

```diff
@@ -0,0 +1,55 @@
+class ChangeLoadBalancerListenerPortToPortRange < ActiveRecord::Migration[5.0]
+  def change
+    reversible do |dir|
+      dir.up do
+        # convert load_balancer_port and instance_port into ranges, preserving
+        # existing values
+        execute <<-SQL
+          ALTER TABLE load_balancer_listeners
+            ALTER COLUMN load_balancer_port SET DATA TYPE int4range
```

Comment:
@selmanj 
or maybe we can use Arel, because you are changing name and type, so you can:
- add new column with range type
- convert data using arel
- remove old column 

@selmanj  what do you think about this solution ?